### PR TITLE
PREAPPS-7943: Implementation | Calendar reminder for important emails

### PR DESCRIPTION
**Pull request by sandesh-telus**
- Modify createAppointment response to use it in zimbra-zimlet-email-reminder

**Related PR:**
- [zm-x-web](https://github.com/Zimbra/zm-x-web/pull/5144)
- [zimlet-cli](https://github.com/Zimbra/zimlet-cli/pull/252)
- [zm-modern-zimlets](https://github.com/Zimbra/zm-modern-zimlets/pull/420)

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -588,7 +588,7 @@ export class ZimbraBatchClient {
 			},
 			accountName,
 			singleRequest: true
-		}).then(Boolean);
+		}).then(res => normalize(CalendarItemCreateModifyRequest)(res));
```
Comment (silentsakky) [inline src/batch-client/index.ts:591]: might be a risky change, please make sure to test Modern Ui core functionality

Comment (sandesh-telus) [reply src/batch-client/index.ts:591]: I have created and modified the event  from the mail and calendar vertical and it has been created successfully.

**Review by miteshsavani810 — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

File: src/batch-client/index.ts (hunk 1 above)
Comment (silentsakky) [reply src/batch-client/index.ts:591]: I guess this should not be required now

Comment (sandesh-telus) [reply src/batch-client/index.ts]: Done. Removed the code.

**Pull request closed without merging**